pkg/apply/path.go

```diff
@@ -0,0 +1,109 @@
+package apply
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"k8s.io/cli-runtime/pkg/genericclioptions"
+)
+
+func processPaths(paths []string) (genericclioptions.FileNameFlags, error) {
```

Comment:
// processPaths attempts to validate its paths argument and return
// an instance of FileNameFlags with validated fields.

(or something)